lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -80,41 +49,29 @@ export default function getRecipeInterface(
     appInfo: NormalisedAppinfo,
     getRecipeImplAfterOverrides: () => RecipeInterface
 ): RecipeInterface {
-    let handshakeInfo: undefined | HandshakeInfo;
-
-    async function getHandshakeInfo(forceRefetch = false): Promise<HandshakeInfo> {
-        if (handshakeInfo === undefined || handshakeInfo.getJwtSigningPublicKeyList().length === 0 || forceRefetch) {
-            let antiCsrf = config.antiCsrf;
-            ProcessState.getInstance().addState(PROCESS_STATE.CALLING_SERVICE_IN_GET_HANDSHAKE_INFO);
-            let response = await querier.sendPostRequest(new NormalisedURLPath("/recipe/handshake"), {});
-
-            handshakeInfo = new HandshakeInfo(
-                antiCsrf,
-                response.accessTokenBlacklistingEnabled,
-                response.accessTokenValidity,
-                response.refreshTokenValidity,
-                response.jwtSigningPublicKeyList
-            );
-
-            updateJwtSigningPublicKeyInfo(
-                response.jwtSigningPublicKeyList,
-                response.jwtSigningPublicKey,
-                response.jwtSigningPublicKeyExpiryTime
+    const JWKS: ReturnType<typeof createRemoteJWKSet>[] = querier.getUrlsForPath("/.well-known/jwks.json").map((url) =>
```

Comment:
We should allow the user to also specify the jwks endpoint cause in m2m case, someone may use our backend sdk without connected to a core and those getSession function calls should verify by querying the apiDomain's jwks endpoint?

This is a very high level thought - to discuss

Reply:
Let's discuss this - but I don't see a significant advantage of always exposing it:
- in the m2m use-case they need the recipe to create the JWTs anyway. 
- in the multiple backends use case (where multiple backends can verify, but only a single backend can create sessions), I think it still makes sense that this endpoint has to be enabled

Reply:
To discuss

Reply:
We decided that we will not do this and instead recommend people to use a standard jwt verification lib with claim checking